index.bs

```diff
@@ -4034,6 +3999,11 @@ the source MUST then output silence (0).
 {{AudioScheduledSourceNode}} cannot be instantiated directly, but
 is instead extended by the concrete interfaces for the source nodes.
 
+An {{AudioScheduledSourceNode}} is said to be <dfn>playing</dfn> when
+its associated {{BaseAudioContext}}'s {{BaseAudioContext|currentTime}} is
+greater or equal to the time it's set to start, and lesser than the time it's
```

Comment:
"it's set to start" -> "AudioScheduledSourceNode is set to start"
"lesser than the time" -> "less than the time"